tests/examples/java_examples_test.go

```diff
@@ -165,6 +165,38 @@ func TestExamples(t *testing.T) {
 		integration.ProgramTest(t, &test)
 	})
 
+	t.Run("minimalsbt", func(t *testing.T) {
+		test := getJavaBase(t, "minimalsbt", integration.ProgramTestOptions{
+			PrepareProject: func(info *engine.Projinfo) error {
+				cmd := exec.Command(filepath.Join(info.Root, "sbt"),
```

Comment:
Is there a reason that `sbt -batch package` command is needed here "out of band"? In the ideal world the Pulumi CLI would talk to the pulumi-language-java language provider, which includes the executor_sbt code, which would pick this up and do what's needed with sbt automagically.

Reply:
I just copied what the other test above does :innocent: 
Should I change it somehow?